src/watchmaker/workers/salt.py

```diff
@@ -523,6 +523,11 @@ def __init__(self, *args, **kwargs):
             kwargs.pop('bootstrap_source', None) or ''
         self.git_repo = kwargs.pop('git_repo', None) or ''
         self.salt_version = kwargs.pop('salt_version', None) or ''
+        self.admin_groups = kwargs.pop('admin_groups', None) or ''
+
+        # Make admin_groups lowercase
+        if self.admin_groups is not None:
```

Comment:
I don't think this test is necessary, since the way it is set/popped above ensures it is always a string.

Reply:
Right.  It'll be a string with values or just an empty string.